src/test/java/carRacing/RacingCarTest.java

```diff
@@ -0,0 +1,33 @@
+package carRacing;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class RacingCarTest {
+
+    RacingCar racingCar;
+
+    @BeforeEach
+    void setUp() {
+        racingCar = new RacingCar();
+    }
+
+    @ParameterizedTest
+    @DisplayName("4이상의 값이면 전진한다")
+    @ValueSource(ints = {4, 5, 6, 7, 8, 9})
```

Comment:
```suggestion
    @ValueSource(ints = {4, 9})
```
굳이 불필요한 Test Fixture를 추가하지 않도록 의식하면 어떨까요?
위 테스트는 경계값이 4, 9면 충분하지 않을까요?